test/functoria/test_impl.ml

```diff
@@ -0,0 +1,22 @@
+(* yiikes *)
+let () =
+  Functoria_misc.Codegen.set_main_ml "main.ml"
+
+let test_keys () =
+  let dev = Functoria.(keys sys_argv) in
+  let info =
+    Functoria.Info.create
+      ~packages:[]
+      ~context:Functoria_key.empty_context
```

Comment:
My memory is a bit fuzzy, but I don't think that should be empty to test what you want to test.